book/src/concepts/system.md

```diff
@@ -134,6 +134,39 @@ This system will make all entities with both a `Transform` with a positive y coo
 
 Cool! Now that looks like something we'll actually do in our games!
 
+### Getting entities that have some components, but not others
+
+There is a special type of `Storage` in specs called `AntiStorage`.
+The not operator (!) turns a Storage into its AntiStorage counterpart.
+It is used like this:
```

Comment:
Just a rewording:
"Sometimes, you may want to iterate over all the entities that do not have a specific component.
You can do this by joining over the `AntiStorage` of your component: the joining will now happen on all entities that do not have the component.
To obtain an `AntiStorage`, simply apply the not operator (!) on your component's normal storage, like so:"